l10n_br_fiscal/models/document.py

```diff
@@ -770,37 +771,40 @@ def _onchange_document_section(self):
             return {'domain': domain}
 
     def _create_return(self):
-        return_ids = self.env[self._name]
+        return_docs = self.env[self._name]
         for record in self:
-            if record.fiscal_operation_id.return_fiscal_operation_id:
-                new = record.copy()
-                new.fiscal_operation_id = (
-                    record.fiscal_operation_id.return_fiscal_operation_id)
-                if record.fiscal_operation_type == 'out':
-                    new.fiscal_operation_type = 'in'
-                else:
-                    new.fiscal_operation_type = 'out'
-                new._onchange_fiscal_operation_id()
-                new.line_ids.write({'fiscal_operation_id': new.fiscal_operation_id.id})
-
-                for item in new.line_ids:
-                    item._onchange_fiscal_operation_id()
-
-                return_ids |= new
-        return return_ids
+            fsc_op = record.fiscal_operation_id.return_fiscal_operation_id
+            if not fsc_op:
+                raise ValidationError(_(
+                    "The fiscal operation {} has no Return Fiscal "
+                    "Operation definied".format(record.fiscal_operation_id)))
```

Comment:
definied -> definied
also be consistent if you use cap letters for Fiscal Operation or not in the same sentence (Odoo uses to do it but not mandatory).